padre_meddea/io/__init__.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from astropy.io import ascii
+from astropy.table import Table
+
+import padre_meddea
+
+detector_values = Table(
```

Comment:
I think the issue with the tests failing is because this is trying to read `detector_values.csv` although it does not exist in the data folder